test/Field.test.tsx

```diff
@@ -121,12 +121,11 @@ describe('Field / FastField', () => {
         </>
       );
 
-      const { handleBlur, handleChange } = getFormProps();
       injected.forEach((props, idx) => {
         expect(props.field.name).toBe('name');
         expect(props.field.value).toBe('jared');
-        expect(props.field.onChange).toBe(handleChange);
-        expect(props.field.onBlur).toBe(handleBlur);
+        expect(typeof props.field.onChange).toBe('function');
+        expect(typeof props.field.onBlur).toBe('function');
```

Comment:
Not the best test ever :)
IMO here we have to test, that calling `field.onChange`/`field.onBlur` actually updates form state.

Reply:
don't do this here, do it in useField or